docs/core/deploying/deploy-with-cli.md

```diff
@@ -29,7 +29,7 @@ Looking for some quick help on using the CLI? The following table shows some exa
 |                                | 3.0 | `dotnet publish -c Release -r <RID> --self-contained true` |
 
 > [!IMPORTANT]
-> \*When using SDK version 3.0, framework-dependent executable this is the default publishing mode when running the basic `dotnet publish` command. This only applies to projects that target **.NET Core 2.1** or **.NET Core 3.0**.
+> \*When using SDK version 3.0, framework-dependent executable is the default publishing mode when running the basic `dotnet publish` command. This only applies when the project targets either **.NET Core 2.1** or **.NET Core 3.0**.
```

Comment:
When you say this only applies, what are you referring to?

Reply:
Oh I see now, you have a \* on the table to correlate to this but this is not very clear.
Maybe it's because is inside an important note?

Reply:
Yeah, the * is hard to see. I removed the note part.